Makefile

```diff
@@ -17,4 +17,18 @@ fmt-js: npm-dependencies
 	npm --prefix ReactApp/ run format
 
 lint-js: npm-dependencies
-	npm --prefix ReactApp/ run lint
\ No newline at end of file
+	npm --prefix ReactApp/ run lint
+
+build_swift_package:
+	xcodebuild build \
+		-scheme GutenbergKit \
+		-sdk iphonesimulator \
+		-destination 'OS=17.5,name=iPhone 15 Plus' \
+		| xcbeautify
+
+test_swift_package:
+	xcodebuild test \
+		-scheme GutenbergKit \
+		-sdk iphonesimulator \
+		-destination 'OS=17.5,name=iPhone 15 Plus' \
+		| xcbeautify
```

Comment:
This `xcodebuild` invocation could be DRYed, and we could have a check for `xcbeautify`

But since I'm not sure whether these commands are needed, I haven't gone down that path yet.